frontend/css/main.less

```
@@ -399,3 +399,8 @@ pre code.hljs {
   padding: 1rem;
   border-radius: 5px;
 }
+
+.feed-button {
+  cursor: pointer;
+  color: #353070;
+}
```

This seems unused, and I think you overwrite the text color on the component using `style`, making it less useful.

I would suggest wrapping the SVG icon inside a parent `div` element with classes `btn btn-icon feed-button`.
The `btn` class handles curser pointer and all that, `btn-icon` makes a button with just the icon as content, and repurposing the `feed-button` class to adjust the padding and size:
```suggestion
.feed-button {
  padding: 0;
  height: auto;
  font-size: 1em;
}
```